Competition2017/tests/competition/subsystems/drive/DriveSubsystemTest.java

```diff
@@ -74,5 +74,12 @@ public void lowPowerInputTankDriveTest(){
         drive.tankDrivePowerMode(0.01, -0.01);
         verifyDriveSetpoints(0.01, -0.01);
     }
+    
+    @Test
+    public void testUpdatePeriodicDataTest(){
```

Comment:
it might be nice to have a test that verifies after shifting the right thing happens in terms of the total distances